exasol/slc/internal/tasks/test/run_db_test_files.py

```diff
@@ -32,21 +31,22 @@ class RunDBTestFiles(
     DatabaseCredentialsParameter,
 ):
 
-    def extend_output_path(self):
-        return self.caller_output_path + ("test_files",)
+    def extend_output_path(self) -> Tuple[str, ...]:
+        return tuple(self.caller_output_path) + ("test_files",)
```

Comment:
Needed explicit `tuple` here, because in ITDE the type hint is wrong: It's `List`, and not `Tuple`.